atlassian/jira.py

```diff
@@ -1109,6 +1104,8 @@ def get_issue(
             params["fields"] = fields
         if properties is not None:
             params["properties"] = properties
+        if expand:
+            params["expand"] = expand
         if update_history is True:
```

Comment:
Would also write:
```python
        params["updateHistory"] = str(update_history).lower()
```
instead of:
```python
        if update_history is True:
            params["updateHistory"] = "true"
        if update_history is False:
            params["updateHistory"] = "false"
```